### PR TITLE
upload service working

### PR DESCRIPTION
**Pull request by dpolozov**
run node awsUpload.js in the services folder

### CODE REVIEW

File: src/components/CreateCampaign.js [hunk 1]
```diff
@@ -214,14 +215,20 @@ class CreateCampaign extends React.Component {
...
+        let newName = `${imgID}.${fileType}`;
+        const formData = new FormData();
+        formData.append(
+            "myFile",
+            this.state.mainImageFile,
+            newName,
+        );
+            
+        return axios.post('http://localhost:3002/awsUpload', formData)
+        .then(res => {
```
Comment (grishick) [inline src/components/CreateCampaign.js:228]: I think this needs exception handling in addition to handling response status codes. E.g., what happens if you pull the plug on the internet when this runs?

File: tsconfig.json [hunk 2]
```diff
@@ -22,5 +22,5 @@
   },
   "include": [
     "src"
-  ]
+, "services/awsUpload.js"  ]
```
Comment (grishick) [inline tsconfig.json:25]: why do we need to include services/awsUpload.js in TSConfig?

**Review by grishick — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (dpolozov) [reply tsconfig.json:25]: the system added it on it's own when i was trying to see the .env files. I think that's when it happened. I didn't actually put than in there myself

Comment (grishick) [reply tsconfig.json:25]: Can you try manually removing it and see if everything still works?

Comment (dpolozov) [reply tsconfig.json:25]: works without it. removed

**Review by grishick — APPROVED** [review]

**Pull request merged**